changes/unreleased/Added-20240808-141816.yaml

```diff
@@ -0,0 +1,6 @@
+kind: Added
```

Comment:
Technical details are not needed on the release note

Reply:
Those are the fields the customer will see in the status, are those the technical details?